kci_rootfs

```diff
@@ -103,7 +103,10 @@ class cmd_build(Command):
         data_path = (
             args.data_path or 'config/rootfs/{}'.format(config.rootfs_type)
         )
-        return kernelci.rootfs.build(config_name, config, data_path, args.arch)
+        if (config.rootfs_type == "chromiumos" and args.board is None):
+            # Set default board
+            args.board = "amd64-generic"
+        return kernelci.rootfs.build(config_name, config, data_path, args)
```

Comment:
How about setting an `board` attribute in the YAML configuration instead? That's what I meant by having separate config entries for each board.

Reply:
Another attribute in the YAML config is probably going to be the chromium branch name, as each board will be using a different one. For the generic ones we could keep using the latest stable release and update it when needed.

Reply:
Done as well